frontend/src/data/test/lightweight_python_functions_v2_pipeline_rev.yaml

```diff
@@ -0,0 +1,236 @@
+pipelineInfo:
```

Comment:
Can you also delete the two json files in https://github.com/kubeflow/pipelines/tree/be0d1dad104ca2ed0e543da3013054ba153180fa/frontend/src/data/test, and add the yaml file for `pipeline_with_loops_and_conditions`